lib/vagrant-libvirt/action/handle_box_image.rb

```diff
@@ -31,18 +126,27 @@ def call(env)
             raise Errors::WrongBoxFormatSet
           end
 
+          env[:box_volume_number] = disks.length()
+          env[:box_volumes] = Array.new(env[:box_volume_number]) {|i| {
+              :path => HandleBoxImage.get_box_image_path(
+                env,
+                disks[i].fetch('path', HandleBoxImage.get_default_box_image_path(i))
+              ),
+              :name => disks[i].fetch('name', HandleBoxImage.get_volume_name(env, i)),
+              :virtual_size => disks[i]['virtual_size'],
+              :box_format => box_format
```

Comment:
I think keep the attribute name as `format` e.g. `:format => box_format`. That way when adding support for different formats on individual disks, more likely to remember to use `:format => disks[i].fetch('format', box_format)`. I do see that will require adjusting the send_box_image and the tests, but I think it'll fit in better with the naming of all the other attributes.

Reply:
I rename `box_format` to `format`.